assets/chezmoi.io/docs/reference/commands/upgrade.md

```diff
@@ -16,3 +16,19 @@ requests should be sufficient for most cases.
 
     If you installed chezmoi using a package manager, the `upgrade` command
     might have been removed by the package maintainer.
+
+## `--executable` *filename*
+
+Set name of executable to replace.
+
+## `--method` *method*
```

Comment:
I do not believe that `--method` should be documented as incorrect selection will cause upgrade failures.

Reply:
I guess then it should be marked as hidden and that should be reflected in documentation as well.

I still think it's better to have this documented to make it explicit that missing documentation is not an overlook. E.g put a warning note that this flag is dangerous and should be used at user's own risk.

Reply:
From the perspective of having supported software for decades: don't document what you don't want people to use. Hidden support flags are *perfectly fine*, and things which should be avoided should be hidden.

Reply:
I would agree with your assessment here except that there *were* changes to the Go code made as part of this PR (making it not just documentation).